handler/src/main/java/io/netty/handler/ssl/DefaultOpenSslApplicationProtocolNegotiator.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import java.util.List;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+/**
+ * OpenSSL {@link ApplicationProtocolNegotiator} for ALPN and NPN.
+ */
+public final class DefaultOpenSslApplicationProtocolNegotiator implements OpenSslApplicationProtocolNegotiator {
```

Comment:
I'm not a fan of having the `Default...` first...groups all the default classes together in the IDE and you have to look deeper to figure out what is actually is.  That being said I'm fine going either way...as long as we are consistent in the package...and hopefully eventually consistent across the entire project :)